src/utils/value_parsers.rs

```diff
@@ -1,8 +1,20 @@
-use anyhow::{anyhow, Result};
+use crate::utils::auth_token::AuthToken;
+use anyhow::{anyhow, Result as AnyhowResult};
```

Comment:
you can continue to use `anyhow::Result` as a drop-in replacement. it has a defaulted type paramater for `E`, but you can use it just the same as `std::Result`.

Reply:
Indeed, I initially implemented this as a closure, where this typing was needed.